apps/src/codebridge/Editor/index.tsx

```diff
@@ -1,25 +1,22 @@
 import {useCodebridgeContext} from '@codebridge/codebridgeContext';
-import {editableFileType} from '@codebridge/utils';
 import {LanguageSupport} from '@codemirror/language';
 import React, {useCallback, useMemo} from 'react';
 
 import {getActiveFileForProject} from '@cdo/apps/lab2/projects/utils';
 import CodeEditor from '@cdo/apps/lab2/views/components/editor/CodeEditor';
 
 import './styles/editor.css';
+import {editableFileType} from '../utils';
 
-export const Editor = (
-  langMapping: {[key: string]: LanguageSupport},
-  editableFileTypes: string[]
-) => {
+interface EditorProps {
+  langMapping: {[key: string]: LanguageSupport};
+  editableFileTypes: string[];
+}
+
+export const Editor = ({langMapping, editableFileTypes}: EditorProps) => {
   const {project, saveFile} = useCodebridgeContext();
 
   const file = getActiveFileForProject(project);
-  // this is a stupid hack. the low level code-mirror editor won't update itself
-  // automatically if the doc is changed externally. So in lieu of doing it better,
-  // for now we just key the component off of the file ID + an incrementing value that
-  // hits every time the format button is pressed. That'll force a re-render and make it work.
-  // let's swap this out with something better.
```

Comment:
I saw this comment recently but couldn't tell what it was referring to exactly. Thanks for cleaning it up!

Reply:
yeah I was pretty sure this was out of date so I removed it, can you confirm @thomasoniii ?

Reply:
yep, totally just out of date and safe to nuke. good catch!